lib/config.js

```diff
@@ -177,6 +177,11 @@ const conf = convict({
       default: 'http://127.0.0.1:9010/v1'
     }
   },
+  customsUrl: {
+    doc: 'fraud / abuse server url',
+    default: 'http://127.0.0.1:7000',
```

Comment:
@larissagaulia this is good. to [follow the auth server configuration](https://github.com/mozilla/fxa-auth-server/blob/master/.env.dev#L2) put `none` into config override https://github.com/mozilla/fxa-profile-server/blob/master/config/dev.json . For your development use `CUSTOMS_SERVER_URL=http://127.0.0.1:7000 npm start` or override dev.json without committing it.